### ISSUE TITLE
V8 Channel Output - LED Panels - Test Pattern

### ISSUE THREAD

**Issue by bobreese** [FalconChristmas/fpp#1976]
Pi4 Gig connection to Colorlight card and P10.
Using the Test Pattern for LED Panels starts the pattern test, when stopping the test, the P10 stops flashing but the test pattern stays on.
Video added to help understand.
[P10Test.zip](https://github.com/user-attachments/files/17019658/P10Test.zip)

**Comment by patdelaney** [comment]
I am able to reproduce this on Pi V8 release.
I am **not** able to reproduce on BB based capes

**Comment by Poporacer** [comment]
I am not able to reproduce on current V8-4. Can you upgrade and see if the problem persists?

**Comment by patdelaney** [comment]
I have 2 pi 3, that I was able to reproduce the issue. Both were fppos upgraded. This morning I burned a fresh image of FPPV8 and tested it and I could hit the test button the test image flashed on the panels and I hit stop and the image disappeared and the panel blanks,  I flipped to Master branch and updatd to latest master and the test button works as expected. 

I'mnot sure where the problem is.
